src/components/pages/room-membership-confirmation.yml

```diff
@@ -1,12 +1,12 @@
+confirmationInProgress:
+  en: Confirmation in progress...
+  de: Confirmation in progress... [in German]
```

Comment:
Would be "Bestätigung in Bearbeitung..." (sounds like lawyer's term though)

As the users already know what process they are currently in, probably a simple "processing..." or "please wait (while we process your request)..." would be enough, I guess?

Reply:
I change it to "Please wait..." / "Bitte warte..." (though not sure if it sounds ok in German)

Reply:
Sounds good